### PR TITLE
WIP: handle nil values when decoding newrelic header

### PR DESCRIPTION
**Pull request by danmo91**

### CODE REVIEW

**Comment by binaryseed** [comment]
Hi there! Can you describe the issue you are seeing?

File: lib/new_relic/distributed_trace/new_relic_context.ex [hunk 1]
```diff
@@ -27,6 +27,7 @@ defmodule NewRelic.DistributedTrace.NewRelicContext do
     end
   end
 
+  def restrict_access(nil)), do: :restricted
```
Comment (danmo91) [inline lib/new_relic/distributed_trace/new_relic_context.ex]:
```suggestion
  def restrict_access(nil), do: :restricted
```

**Comment by binaryseed** [comment]
I think this addresses what you were seeing:

https://github.com/newrelic/elixir_agent/pull/190

**Comment by danmo91** [comment]
Thank you!! That solved it for me 🙇‍♂️

**Pull request closed without merging**